tests/snappi/test_vlan_traffic.py

```diff
@@ -0,0 +1,140 @@
+import pytest
+from pprint import pprint as pprint
+import time
+
+vlan_id = "10"
+macs = ["00:1A:C5:00:00:01", "00:1B:6E:00:00:01"]
+
+
+@pytest.fixture(scope="module", autouse=True)
+def discovery(npu):
+    npu.objects_discovery()
+
+
+def test_l2_untagged_vlan_traffic(npu, dataplane):
```

Comment:
Please make sure this TC will we skipped in case pytest is executed without specifying the test file of or tests wildcard and dataplane is not snappi.

Reply:
added the skip all fixture

Reply:
As I mentioned above, please make sure that the traffic related part of TC will be skipped in case dataplane type is not `snappi`.
```
if npu.run_traffic and dataplane.config["type"] == "snappi":
    # Traffic check
```